config/http.conf

```diff
@@ -2,12 +2,17 @@ http {
   include /etc/nginx/mime.types;
   include /etc/nginx/log.conf;
 
-  server_tokens         off;
-  keepalive_timeout     20s;
-  sendfile              on;
-  tcp_nopush            on;
-  client_max_body_size  400m;
-  client_body_timeout   300s;
+  server_tokens           off;
+  sendfile                on;
+  tcp_nopush              on;
+
+  client_max_body_size    500m;
+  client_body_buffer_size 128k;
+  client_body_timeout     300s;
+  client_header_timeout   65s;
+  keepalive_timeout       65s;
+  proxy_read_timeout      600s;
```

Comment:
To note, we should probably think about all these:

```
proxy_connect_timeout 600;
proxy_send_timeout 600;
proxy_read_timeout 600;
```

Reply:
If the ALB is 600s this should be greater, right?

Reply:
Not really, this proxy configuration so they would apply to the backend i.e. target of `proxy_pass` for example. I think you're thinking about `keepalive_timeout` and `client_header_timeout`.

Reply:
Discussed privately, agreed to set it to `605s` to allow an ELB up to `10m` of idle time. See https://github.com/Intellection/docker-nginx-proxy/commit/a58861c42608a0eef0159e5bf4722ee7e96997b7.